VideoLocker/src/main/java/org/edx/mobile/view/adapters/DiscussionCommentsAdapter.java

```diff
@@ -27,9 +29,9 @@
         void reportComment(DiscussionComment comment);
     }
 
-    @Inject
-    public DiscussionCommentsAdapter(Context context, Listener listener, DiscussionComment response) {
+    public DiscussionCommentsAdapter(Context context, Router router, Listener listener, DiscussionComment response) {
```

Comment:
The `Router` instance can be injected here. It doesn't need to be part of the parameters.